src/test/java/uk/gov/companieshouse/service/dissolution/DissolutionPatcherTest.java

```diff
@@ -10,6 +10,7 @@
 import uk.gov.companieshouse.exception.DissolutionNotFoundException;
 import uk.gov.companieshouse.fixtures.DissolutionFixtures;
 import uk.gov.companieshouse.mapper.DirectorApprovalMapper;
+import uk.gov.companieshouse.mapper.DissolutionDirectorResponseMapper;
```

Comment:
does this file need reverted?

Reply:
Since I touched the file already, sonar will complain about "public" modifiers for test classes. So I only lefte these changes and removed the others.